ironic-config/httpd-ironic-proxy.conf.j2

```diff
@@ -0,0 +1,49 @@
+ServerRoot {{ env.HTTPD_DIR }}
+Listen [::]:{{ env.IRONIC_PROXY_PORT }}
```

Comment:
I have removed the `LISTEN_ALL_INTERFACES` if-block and directly added this lines based on what we were doing in runironic-proxy `sed` statement. Does that make sense ?
And, will `Listen {{ env.IRONIC_PROXY_PORT }}` be more appropriate ?

Reply:
Also, note that I have replaced `HTTP_PORT` with `IRONIC_PROXY_PORT`.